src/deadlock.rs

```diff
@@ -0,0 +1,195 @@
+#[cfg(feature = "deadlock_detection")]
+pub use parking_lot_core::deadlock::{acquire_resource, release_resource};
+
+#[cfg(not(feature = "deadlock_detection"))]
+#[inline]
+pub fn acquire_resource(_key: usize) {
+    // noop
+}
+
+#[cfg(not(feature = "deadlock_detection"))]
+#[inline]
+pub fn release_resource(_key: usize) {
+    // noop
+}
```

Comment:
I meant that you should make these functions no-ops in `parking_lot_core`, not here.

Reply:
Ahh, makes sense.